src/components/Search/SearchFiltersParticipantsSelector.tsx

```diff
@@ -104,19 +104,20 @@ function SearchFiltersParticipantsSelector({initialAccountIDs, onFiltersUpdate}:
             true,
         );
 
-        const isCurrentUserSelected = selectedOptions.find((option) => option.accountID === chatOptions.currentUserOption?.accountID);
+        const currentUser = formattedResults.section.data.find((option) => option.accountID === chatOptions.currentUserOption?.accountID);
 
-        newSections.push(formattedResults.section);
-
-        if (chatOptions.currentUserOption && !isCurrentUserSelected) {
+        if (chatOptions.currentUserOption) {
             const formattedName = ReportUtils.getDisplayNameForParticipant(chatOptions.currentUserOption.accountID, false, true, true, personalDetails);
-            newSections.push({
-                title: '',
-                data: [{...chatOptions.currentUserOption, text: formattedName}],
-                shouldShow: true,
-            });
+            if (!currentUser) {
+                chatOptions.currentUserOption.text = formattedName;
+                chatOptions.recentReports.unshift(chatOptions.currentUserOption);
```

Comment:
I dislike unshift, remember you can always do:

```
const oldArray = [...];
const newArray = [item, ...oldArray];
```

(PS. nothing is copied in the code above, merely a reference to an array, so its not heavy)